cocinaborcelle/src/components/CardCompras.tsx

```diff
@@ -0,0 +1,49 @@
+
+import { useState } from 'react';
+import styles from "@/styles/CardCompras.module.css";
+
+type CardComprasProps = {
+    title?: string;
+    number: number;
+}
+
+export default function CardCompras({ title }: CardComprasProps) {
+    const [count, setCount] = useState(0);
```

Comment:
```suggestion
    const [count, setCount] = useState(compra.quantidade);
```

passar para count a quantidade da compra